app/main.py

```diff
@@ -115,6 +115,7 @@ def main():
              This tool brings together feedback from users submitted via the \
              'report a problem on this page' link and smart survey responses."
     )
+    st.text("")
```

Comment:
Worthwhile flagging that in previous projects I used:

```
def space(num_lines=1):
    """Adds empty lines to the Streamlit app."""
    for _ in range(num_lines):
        st.write("")
```

Doesn't make a difference here but does if you need to do multiple instances of blank space